ci/check_diff.sh

```diff
@@ -43,7 +43,7 @@ function init_submodules() {
 # $2: Output file path for the diff
 # $3: Any additional configuration options to pass to rustfmt
 #
-# Globlas:
+# Globals:
```

Comment:
this is by far the most worrisome of the new changes

Reply:
Wondering if you can elaborate on why this is worrisome? For context, this is a script that occasionally gets run in CI.

Reply:
It's consistently spelled this way, so for all I know it's a real thing :)

Hopefully it's just a consistent typo.

Reply:
Yeah, I misspelled it, and then I copy pasted it 😅 that's on me